src/latexify/codegen/function_codegen.py

```diff
@@ -355,6 +355,30 @@ def visit_Call(self, node: ast.Call) -> str:
             (r"\mathrm{" + func_str + r"}\left(", r"\right)"),
         )
 
+        if func_str in ("ndarray", "array"):
+            arg = node.args[0]
+            if not isinstance(arg, ast.List) or not arg.elts:
+                return None
+
+            row0 = arg.elts[0]
+
+            if not isinstance(row0, ast.List):
+                return self._generate_ndarray([self.visit(x) for x in arg.elts])
+
+            if not row0.elts:
+                return None
+
+            nCols = len(row0.elts)
+
+            if not all(
+                isinstance(row, ast.List) and len(row.elts) == nCols for row in arg.elts
+            ):
+                return None
+
+            return self._generate_ndarray(
+                [[self.visit(x) for x in row.elts] for row in arg.elts]
+            )
```

Comment:
This block must be placed on _generate_ndarray, and if it returns None we need to fall back to the default behavior.